Misc/stable_abi.toml

```diff
@@ -2462,3 +2462,7 @@
     added = '3.13'
 [function.Py_IsFinalizing]
     added = '3.13'
+[function.PyUnicode_EqualToUTF8]
+    added = '3.13'
+[function.PyUnicode_EqualToUTF8AndSize]
+    added = '3.13'
```

Comment:
Unrelated question, but is there a plan to generate this file from `Doc/data/stable_abi.dat` or the reverse?

Reply:
I think that `Doc/data/stable_abi.dat` is generated from `Misc/stable_abi.toml`.

Reply:
Ah, ok, thank you!